rasa/nlu/classifiers/__init__.py

```diff
@@ -1,3 +1,3 @@
 # How many labels are at max put into the output
 # ranking, everything else will be cut off
-LABEL_RANKING_LENGTH = 10
+DEFAULT_LABEL_RANKING_LENGTH = 10
```

Comment:
let's use customizable `ranking_length = 10` in both nlu and core algorithms and keep this constant as maximum ranking length in the output of nlu classifier, if `ranking_length` is set to `0` or `-1`

Reply:
also add to response selector

Reply:
`-1` or `0` should be no normalization

Reply:
right now it only normalizes if ranking_length > 0, so both -1 and 0 should work